### DISCUSSION TITLE
Using TestSolvingLinearPdesTutorial

### DISCUSSION THREAD

**Discussion by lolsenkettle** [Chaste/Chaste#309] [Q&A] [answered]
I am able to run the TestSolvingLinearPdesTutorial from my chaste build directory. 

However when I make a copy of the code in my project repository (downloaded as per instructions on github), and try to run the same code I get the following error: 
![Screenshot from 2024-09-05 13-41-33](https://github.com/user-attachments/assets/2f1566d1-729a-4fcc-8fb0-ea579a5ae7e0)

Can you tell me how I can add the SimplePoissonEquation.hpp and HeatEquationWithSourceTerm.hpp files to my user project and then add to my ContinuousTestPack.txt file so that the example can be run and modified?

**Comment by jmosborne** [comment, marked as answer]
When you are using a user project it only include files from pde/src not
pde/test which is where the SimnplePoissonEquation files are.

Try changing that line to

#include <pde/test/pdes/SimplePoissonEquation.hpp>

James

**Reply by lolsenkettle** [reply]
Thanks James. I got it to work by adding those files to my project repository as well. 
To visualise the results are we able to use >SetVisualizeWithCmgui(true) or SetVisualizeWithVtk(true) with this tutorial example? I tried using this: solver.SetVisualizeWithVtk(true);  but got this error:

class SimpleLinearParabolicSolver<3, 3>’ has no member named ‘SetVisualizeWithVtk’
solver.SetVisualizeWithVtk(true);


or do we need to write the mesh to a file using:
        TrianglesMeshWriter<3,3> mesh_writer("ParabolicSolverTutorial2", "mesh", false /*don't clean (ie delete everything in) directory!*/);
        mesh_writer.WriteFilesUsingMesh(mesh);

**Reply by MILeach** [reply]
Hi @lolsenkettle, we saw that you've opened another discussion since you opened this one - I just wanted to check that you've solved this problem and whether we could close this discussion now?

**Reply by lolsenkettle** [reply]
Yes it is working now. Thanks.
